bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemUtil.java

```diff
@@ -39,8 +39,8 @@ private ItemUtil() {
      *
      * @return true if the specified name is a valid item name, otherwise false
      */
-    public static boolean isValidItemName(String itemName) {
-        if ((itemName != null) && (!itemName.isEmpty())) {
+    public static boolean isValidItemName(final String itemName) {
```

Comment:
Could catch to use  the assert... one in AbstractLink.
Will change that.

> Why do we need this method?

I don't know. As you could see, this method has NOT introduced by this commit. It exists already.